packages/flutter_tools/lib/src/isolated/web_asset_server.dart

```diff
@@ -237,6 +239,12 @@ class WebAssetServer implements AssetReader {
     // Allow rendering in a iframe.
     httpServer!.defaultResponseHeaders.remove('x-frame-options', 'SAMEORIGIN');
 
+    if (crossOriginIsolation) {
+      for (final MapEntry<String, String> header in kMultiThreadedHeaders.entries) {
```

Comment:
Shall this be renamed `kMultiThreadedHeaders` => `kCrossOriginIsolationHeaders` for consistency?